product/nomad.go

```diff
@@ -23,10 +24,20 @@ func NewNomad(cfg Config) (*Product, error) {
 		return nil, err
 	}
 
+	// Apply nomad duration and interval default if CLI is using global defaults
+	// NOTE(mkcp): This isn't ideal because we're using magic numbers here to match the CLI defaults. We could pass a
+	//  tuple or something in from the CLI that describes both the default and the user-set value... but i'm timeboxing this.
+	if cfg.DebugDuration == 10*time.Second {
+		cfg.DebugDuration = NomadDebugDuration
```

Comment:
Not necessarily for this PR, but I wonder if it would be useful to pull some of the defaults out of the `main` package and into products. `main.TenSeconds` could become, for example, `product.DefaultDebugDuration`. `main.go` could then import this to insert into the help text, but we could then do some checks within product (such as here, with Nomad), where we could more easily check wither, for e.g., `if cfg.DebugDuration == DefaultDebugDuration {}`.